### PR TITLE
Toast component

### PR DESCRIPTION
**Pull request by mehulsatardekar**
Added Toast component

Live preview :https://framer-ui.netlify.app/docs/toast.html

### CODE REVIEW

File: components/toast/toast.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
+/* @import url('../../components/root.css');
+@import url('../../components/Buttons/button.css');
+@import url('../../components/alert/alert.css'); */
```
Comment (abhijeetekad) [inline components/toast/toast.css:3]: Remove this commented code.

File: components/toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+}
+
+.snackbar-icons{
+    font-size: 20px !important;
+    padding: 0.3rem;
+    cursor: pointer;
+
+}
+.snackbar-icons:hover{
+    background: rgba(151, 153, 155, 0.411);
```
Comment (abhijeetekad) [inline components/toast/toast.css:42]: Use can colour variables here also.

File: docs/toast.html [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+                        id="toastExampleCenter">
+                        <span>This item already has the label "travel". You can add a new label.</span>
+                        <div class=" flex flex-justify-end  flex-align-item-center gap mt-1">
+                            <span>Undo</span>
+                            <span class="material-icons  snackbar-icons" id="toastExampleCenterCloseBtn">close</span>
+                        </div>
+                    </div>
+
+
+                    </script>
```
Comment (abhijeetekad) [inline docs/toast.html:245]: I think this is of no use. you can remove this.

**Review by abhijeetekad — APPROVED** [review]
Do the suggested changes and then good to merge

File: docs/toast.html (hunk 3 above)
Comment (mehulsatardekar) [reply docs/toast.html:245]: > I think this is of no use. you can remove this.

it's a closing tag to wrap code inside a script (Just a PrismJs Thing). I have wrapped all the HTML code inside this script tag.

Comment (abhijeetekad) [reply docs/toast.html:245]: Ohh.. then it's ok..

**Review by KanekarSnehal — APPROVED** [review]
Well structured, you can merge it.

**Pull request merged**